packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -701,13 +701,11 @@ export class PeerManager {
    * dialed or connecting to us.
    */
   private onLibp2pPeerConnect = async (evt: CustomEvent<Connection>): Promise<void> => {
-    const {direction, status, remotePeer} = evt.detail;
+    const {direction, remotePeer} = evt.detail;
```

Comment:
Destructuring the `status` will not reflect any change that happens on connection during execution of this function.